### PR TITLE
Update to Minecraft 1.21

### PR DESCRIPTION
**Pull request by DaddyWesker**
Subj

### CODE REVIEW

**Comment by noskill** [comment]
@DaddyWesker Could you please update launch.sh for tests?

**Comment by DaddyWesker** [comment]
I'll do my best.

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java [hunk 1]
```diff
@@ -4,9 +4,20 @@
...
+import net.minecraft.registry.*;
+import net.minecraft.registry.entry.RegistryEntryInfo;
+import net.minecraft.util.ErrorReporter;
+import net.minecraft.util.Util;
+import net.minecraft.util.math.random.RandomSeed;
+import net.minecraft.util.math.random.RandomSequence;
 import org.jetbrains.annotations.Nullable;
-import net.minecraft.loot.LootDataKey;
-import net.minecraft.loot.LootDataLookup;
+//import net.minecraft.loot.LootDataKey;
```
Comment (noskill) [inline src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java]:
```suggestion
```

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java [hunk 2]
```diff
@@ -4,9 +4,20 @@
...
+//import net.minecraft.loot.LootDataLookup;
```
Comment (noskill) [inline src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java]:
```suggestion
```

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java [hunk 3]
```diff
@@ -215,52 +231,57 @@ else if (predicate.toString().contains("items"))
...
 
-        while (var5.hasNext()) {
-            Identifier identifier = (Identifier) var5.next();
-            lootTableReporter.report("Missing built-in table: " + identifier);
+        mutableRegistry.freeze();
+        ErrorReporter.Impl impl = new ErrorReporter.Impl();
+        RegistryEntryLookup.RegistryLookup registryLookup2 = (new DynamicRegistryManager.ImmutableImpl(List.of(mutableRegistry))).toImmutable().createRegistryLookup();
+        LootTableReporter lootTableReporter = new LootTableReporter(impl, LootContextTypes.GENERIC, registryLookup2);
+        Iterator var9 = Sets.difference(lootTableIds, mutableRegistry.getKeys()).iterator();
+
```
Comment (noskill) [inline src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java]:
```suggestion
// todo: add a comment indicating where this decompiled code was adapted from
```

**Pull request closed without merging**